src/test/java/com/internship/backend/controller/AuthorityControllerTest.java

```diff
@@ -0,0 +1,110 @@
+package com.internship.backend.controller;
+
+import com.internship.backend.dto.AuthorityDTO;
+import com.internship.backend.exceptions.UserDoesNotExistException;
+import com.internship.backend.model.Authority;
+import com.internship.backend.service.AuthorityService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.List;
+
+import static org.hamcrest.Matchers.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+class AuthorityControllerTest {
+
+    @InjectMocks
+    private AuthorityController authorityController;
+
+    @Mock
+    private AuthorityService authorityService;
+
+    private MockMvc mockMvc;
+    private Authority authority;
+    private AuthorityDTO authorityDTO;
+    private List<Authority> authorities;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+        mockMvc = MockMvcBuilders.standaloneSetup(authorityController).build();
+
+        authority = Authority.builder()
+                .id(1)
+                .name("ROLE_USER")
+                .build();
```

Comment:
If this is the default authority used on test class, it can be extracted in a private method to avoid polluting the setUp method